src/UI/templates/default/Input/Rating/rating.less

```diff
@@ -0,0 +1,63 @@
+
+
+.il-input-rating-topic{
+	 float: left;
+	 margin-right: 5px;
+	}
+
+.il-input-rating {
+	display: inline-block;
+
+	input {
+		float: right;
+		padding: 0;
+		margin: 0 0 0 -20px;
+		opacity: 0;
+	}
+
+	/*delos uses "\e006", which is the right-arrow...*/
+	.glyphicon-star:before {
+	  content: "\2605" !important;
+	}
+
+	.glyphicon-star {
+		float: right;
+		position: relative;
+		display: block;
+		margin-top: -8px;
+		margin-bottom: 0px;
+		font-size: 160%;
+		color: @il-glyph-color;
+		/*
+			for disabled mode, use:
+			color: @il-glyph-hover-color;
+		*/
```

Comment:
Inputs should have a disabled-status, i.e. readonly, but showing values. 
This is not implemented; I just pinned a note for the colors I thought to be the ones to use.